integration_tests/test_sage_detection.py

```diff
@@ -64,12 +64,16 @@ def test_create_asset_group_detection(session, codex_url, test_root, login):
         ],
         'completion': 10,
         'stage': 'curation',
+        'locationId': 'Tiddleywink',
+        'time': '2000-01-01T01:01:01+00:00',
+        'timeSpecificity': 'time',
         'jobs': [
             {
                 'job_id': first_job['job_id'],
                 'model': 'african_terrestrial',
                 'active': False,
                 'asset_ids': [asset_guid],
+                'start': first_job['start'],
```

Comment:
why this change means that the job is started by the time this message is received is a bit of a mystery.